.rubocop_fixme.yml

```diff
@@ -23,7 +23,7 @@ Metrics/AbcSize:
 
 Metrics/ModuleLength:
   Exclude:
-    - 'app/controllers/concerns/hyrax/collections_controller_behavior.rb'
+    - 'app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb'
```

Comment:
My answer about using `/dashboard/` was about the routes/URLs. (We don't see `/admin/` in many routes.) @jcoyne's point is about module namespaces.